content/faq/android-fdroid.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "F.A.Q. - Android"
+description: "Frequently asked questions for Organic Maps application"
+weight: 1000
+
+taxonomies:
+  faq: ["Android"]
+
+extra:
+  question: What's the difference between F-Droid and Google Play versions of tha app?
```

Comment:
What about moving the question to the title? It would be easier to support, and should improve SEO.

Reply:
Moved to title.